dashboard/src/reducers/namespace.ts

```diff
@@ -32,7 +32,8 @@ const namespaceReducer = (
     case getType(actions.namespace.errorNamespaces):
       return { ...state, errorMsg: action.payload.err.message };
     case getType(actions.namespace.clearNamespaces):
-      return { ...initialState };
+      // Clear namespaces info but keep "current" to avoid unexpected redirections
+      return { ...initialState, current: state.current };
```

Comment:
> After login out the current namespace was cleared to the default value (default). This caused that after login out and in again the namespace changed from whatever it was to default.

Isn't that the expected behaviour - that if I log out there is no state, so that when I log back in, I'll be put on the default namespace? I'm not sure what I'm missing on this one, but +1'ing so we can start those release images :)

Reply:
not really, the state it's better cleaned up when reloading the page.

Let me put an example of the issue.

1.  I got my token for the namespace `user-1`. I login in Kubeapps with that token and go to to my namespace. 
2. I go to the `/catalog` page, which doesn't have the namespace in the route.
3. I click on "logout" and the current namespace is set back to `default`.
4. I login again, and since I haven't changed the namespace in the selector I got again in the `/catalog` page again but this time, the selected namespace is `default`.